tensorflow_addons/callbacks/tqdm_progress_bar.py

```diff
@@ -168,10 +168,9 @@ def _update_progbar(self, logs):
 
         self.num_samples_seen += batch_size
         self.steps_to_update += 1
-        self.steps_so_far += 1
-
-        if self.steps_so_far < self.total_steps:
 
+        if self.steps_so_far <= self.total_steps:
```

Comment:
Tiny NIT: actually we should either change `<` to `<=` OR putting `self.steps_so_far += 1` inside but not both, this will cause to take one extra step (this should not cause error anyway since `on_epoch_end` wouldn't be called the extra time for this to occur, but for readability may you move `self.steps_so_far += 1` back before, I think changing `<` to <=` alone should solve it, thanks!

Reply:
@shun-lin , changed this tiny error. Thanks for noticing. Please approve it again.